change/@fluentui-react-icons-mdl2-branded-2021-01-27-15-29-05-react-icons-mdl2-branded-package.json

```
@@ -0,0 +1,8 @@
+{
+  "type": "none",
```

Otherwise it won't get published
```suggestion
  "type": "prerelease",
```